index.html

```diff
@@ -72,5 +88,183 @@
 	  	
 	</div>
 </section>
+
+<section id="body">
+	<div class="flex-body">
+		<div class="recent-work">
+			My Recent Works
+			<hr>
+		</div>
+		
+		<div class="stories">
+			<img src="assets/images/placeholder-1.png" alt="stories">
+			<div class="work-sample-2"> Multy-Post Stories </div>
+			<div class="work-sample-detail">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+			</div>
+			<div class="tags-2">
+				<div class="tag">css</div>
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
```

Comment:
- [ ] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.